diff --git a/include/config.h b/include/config.h
index 0a45ae0..a314e85 100644
--- a/include/config.h
+++ b/include/config.h
@@ -47,13 +47,13 @@ THE SOFTWARE.
 	#define CAN_S_Pin GPIO_PIN_13
 	#define CAN_S_GPIO_Port GPIOC
 
-	#define LED1_Pin GPIO_PIN_0
+	#define LED1_Pin GPIO_PIN_1
 	#define LED1_Mode GPIO_MODE_OUTPUT_OD
 	#define LED1_GPIO_Port GPIOA
 	#define LED1_Active_High 0
 
 	#define LED2_GPIO_Port GPIOA
-	#define LED2_Pin GPIO_PIN_1
+	#define LED2_Pin GPIO_PIN_0
 	#define LED2_Mode GPIO_MODE_OUTPUT_OD
 	#define LED2_Active_High 0
 
